galera/Project1/Galera.h

```diff
@@ -1,20 +1,27 @@
 #pragma once
 #include<string>
+#include"Employee.h"
+#include<vector>
 
 using namespace std;
 
 class Galera
 {
 private:
     string _name;
-    int _EmployeeCount;
+    int _employeeCount;
     int _totalSallary;
+    vector<Employee> _employeeList;
 
 public:
-    void setName(string name);
+    Galera(string name);
     string getName() const;
     void setEmployeeCount(int count);
     int getEmployeeCount() const;
     void setTotalSallary(int total);
     int getTotalSallary() const;
-};
+    bool hireEmployee(Employee);
```

Comment:
In this pull request we should have a fix for `setName` issues. Why do we have here `Galera::hireEmployee` and `Developer::setLanguage`? Please, create one pull request for one issue in the future